pysoa/server/server.py

```diff
@@ -690,7 +690,7 @@ def harakiri(self, signal_number, _stack_frame):  # type: (int, FrameType) -> No
             threads.setdefault(thread_id, {'name': thread_id})['traceback'] = stack
 
         extra = {'data': {'thread_status': {
-            t['name']: [l.rstrip() for l in t['traceback']] for t in threads.values()  # noqa: E741
```

Comment:
I found this lint error, and I fixed it. I didn't like to omit the error by adding the exception # noqa